docs/background/project-del-not.md

```diff
@@ -0,0 +1,44 @@
+# Why you should not bother deleting projects
+
+You may have noticed that the {{gui}} does not provide a mechanism
+for completely wiping out projects, also known as _tenants_. In other
+words, you cannot simply delete a project and **all** of its resources
+in one go. Also, you cannot delete a project even after you have
+deleted all of its resources. While this operation is implemented,
+it is reserved for domain admins only, and {{brand}} users have no
+domain admin privileges.
+
+Following is a list of facts explaining our policy **against** wiping
+out or simply deleting projects.
+
+## Wiping out a project is not possible
```

Comment:
I chose "wipe out" as a quite dramatic term to describe destroying a project along with any resources it may have. But "purge" is also dramatic (and a bit more descriptive), so let's go with that.